domain/domain.go

```diff
@@ -142,20 +138,26 @@ func (do *Domain) tryReload() {
 	}
 }
 
-func (do *Domain) reload() error {
+func (do *Domain) reload(retCh chan error) error {
 	err := kv.RunInNewTxn(do.store, false, do.loadInfoSchema)
```

Comment:
do the timeout check in reload, not outer. 
not only ticker but also others can call reload too.